release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -934,6 +934,17 @@ BMER was deprecated in {product-title} version 4.15 and 4.16. With this release,
 [id="ocp-4-17-image-registry-bug-fixes_{context}"]
 ==== Image Registry
 
+* In {product-title} 4.14, installing a cluster with Azure AD Workload Identity was made generally available. With that feature, administrators can configure a Microsoft Azure cluster to use Azure AD Workload Identity. With Azure AD Workload Identity, cluster components use short-time security credentials that are managed outside of the cluster. 
++
+Before this update, when {product-title} was deployed on Azure clusters with Azure AD Workload Identity, storage accounts that were created for the cluster and the image registry had storage account key access enabled by default. The use of storage account keys like Shared Access Signatures (SAS) could create security risks for some {product-title} on Azure deployments.
```

Comment:
I also confuse SAS with shared access key, but I think access keys are their own thing. In fact, I think that we are not getting rid of SAS because pulls from registry by default will redirect the client to pull the image from the storage directly, and for that to work SAS is required.

What we're disabling here (in clusters configured with workload identity) is shared access keys.

In case it helps, here are the azure docs for shared access keys: https://learn.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal.
And here are the docs for SAS: https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview

@rajdeepc2792 can keep me honest here.